lib/postgrex/protocol.ex

```diff
@@ -347,8 +350,17 @@ defmodule Postgrex.Protocol do
     status = new_status(opts, prepare: prepare)
 
     case prepare do
-      true -> parse_describe_close(s, status, query)
-      false -> parse_describe_flush(s, status, query)
+      true ->
+        parse_describe_close(s, status, query)
+
+      false ->
+        comment = Keyword.get(opts, :comment)
+
+        if is_binary(comment) && String.contains?(comment, "*/") do
```

Comment:
```suggestion
        if is_binary(comment) and String.contains?(comment, "*/") do
```

Reply:
Let's move this validation to the client. There is no need to crash the connection on bad argument. :)

Reply:
So should I just remove the whole `if` here?

Reply:
You should move it to the client, to the `Postgrex` module.